var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -86,6 +87,7 @@ class Warpx(CMakePackage):
     depends_on('lapackpp', when='+psatd dims=rz')
     depends_on('mpi', when='+mpi')
     depends_on('openpmd-api@0.13.1:', when='+openpmd')
```

Comment:
You can now do

```
with when('+openpmd'):
    depends_on('openpmd-api@0.13.1:')
    depends_on('openpmd-api@0.14.2:', when='@21.09:')
    depends_on('openpmd-api ~mpi', when='~mpi')
    depends_on('openpmd-api +mpi', when='+mpi')
```

if that makes it easier to follow?

Reply:
Oh cool! Will update next time :+1:

Reply:
->  #25789 :)